lib/js/components/Statuses/BlockadeStatus/BlockadeStatus.vue

```diff
@@ -9,20 +9,27 @@
 @import '../../../../styles/settings/colors';
```

Comment:
why don't you remove this import?

Reply:
Good catch, I'm removing all usages. There was only problem in `Modal.vue` - it helps to catch usage old colors.